docs/docs-beta/docs/guides/data-modeling/configuring-assets.md

```diff
@@ -1,5 +1,40 @@
 ---
-title: Configuring assets and ops
-sidebar_label: Configuring assets
+title: Configuring assets and ops via the UI
```

Comment:
```suggestion
title: Configuring runs via the UI
```

Reply:
```suggestion
title: Configuring assets and ops using the Dagster UI
```

Nit - we're moving away from `via` (see the Vale rule) and when referring to the UI in a page title, it should be the complete `Dagster UI`